include/arch/arm/arch/machine/gic_common.h

```diff
@@ -37,8 +37,8 @@
  * The layout of IRQs into the CNode are all of PPI's for each core first, followed
  * by the global interrupts.  Examples:
  *   core: 0, irq: 12 -> index 12.
- *   core: 2, irq: 16 -> (2 * 32) + 12
- *   core: 1, irq: 33, (4 total cores) -> (4 * 32) + (33-32).
+ *   core: 2, irq: 16 -> (2 * 32) + 16
+ *   core: 1, irq: 33, (4 total cores) -> (3 * 32) + 33.
```

Comment:
Sees the `(33-32)` comes from seeing it as `(CONFIG_MAX_NUM_NODES * NUM_PPI + (_irq.irq - NUM_PPI))`, but at some point this was simplified to `(CONFIG_MAX_NUM_NODES - 1) * NUM_PPI + (_irq.irq)` then in the code.

Reply:
Axel's right that the implementation was simplified.  The example is trying to explain the design and should be illustrating that the SPI interrupts start from 0 at the end of the PPI/SGI of all the cores.